src/util/Injected.js

```diff
@@ -153,6 +155,57 @@ exports.LoadUtils = () => {
         return window.Store.Msg.get(newMsgId._serialized);
     };
 
+    window.WWebJS.sendImageAsSticker = async function (mediaInfo, chatId) {
+        const stickerInfo = await window.WWebJS.processStickerData(mediaInfo);
+        return await window.WWebJS.sendSticker(stickerInfo, chatId);
+    };
+
+    window.WWebJS.sendSticker = async function (stickerInfo, chatId) {
```

Comment:
I would rename this to sendStickerRaw, as there still is potential to add a sendSticker via Sticker Packs which is not yet included in this PR.